trimmer.rb

```diff
@@ -0,0 +1,7 @@
+require_relative './decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable[0, 10]
```

Comment:
- [ ] The should have a maximum of 10 characters. If it's longer it should trim the word.
- [ ]  The code is a decorator that will correct the name of an object.
 The code first checks to see if the length of the string is greater than 10, and then it returns whatever number is in between 0 and 9.
 The code will return the correct name of the person's first and last names.
 The code will return a string with no spaces in it

Reply:
thanks for the clarifications @dasileker, I get it 👍🏾  😄